src/main/java/org/apache/ibatis/executor/resultset/DefaultResultSetHandler.java

```diff
@@ -477,7 +477,7 @@ private Object getPropertyMappingValue(ResultSet rs, MetaObject metaResultObject
           }
         }
         final String property = metaObject.findProperty(propertyName, configuration.isMapUnderscoreToCamelCase());
-        if (property != null && metaObject.hasSetter(property)) {
+        if (property != null && metaObject.hasSetter(property) && !resultMap.getMappedProperties().contains(property)) {
```

Comment:
Hi @harawata 

In My opinion, the `phone` is valid column name as auto-mapping (if does not use explicit mapping...), it skip because it has been mapped by explicit mapping already. Hence i think `phone` is not unknown column.
I think we should consider to apply a mechanism that notify duplicate mapping for same property (like same as autoMappingUnknownColumnBehavior).

What do you think ?